docs/self-hosting/configuration/domain-configuration.mdx

```diff
@@ -27,36 +27,165 @@ NEXTAUTH_URL=https://formbricks.example.com
+- **SDK Integration**: The public domain is used for all client-side SDK connections
+- **Email Links**: Survey invitation emails use the public domain
+- **Sharing Features**: Survey sharing uses the public domain
+
+### Routes Available on the Public Domain
+
+When PUBLIC_URL is configured, the following routes are automatically served from the public domain:
+
+#### Survey Routes
+
+- `/s/{surveyId}` - Individual survey access
+- Embedded survey endpoints
+
+#### API Routes
+
+- `/api/v1/client/{environmentId}/*` - Client API endpoints
+
+#### Storage Routes
+
+- `/storage/{environmentId}/{accessType}/{fileName}` - Public file storage access
+
+#### Health & Monitoring
+
+- `/health` - Health check endpoint
+
+#### Sharing & Embed Routes
+
+- Survey sharing endpoints
+- Embed widget routes
+- Social sharing endpoints
+
+All other routes are restricted to the private domain (`WEBAPP_URL`) for security purposes. The admin domain can access the health endpoint (/health) and public file storage (`/storage/{environmentId}/{accessType}/{fileName}`), while all other public routes are exclusively accessible on the public domain (`PUBLIC_URL`) and will return a 404 error if accessed from the admin domain.
+
+### App URL SDK Changes
+
+The domain separation introduces important changes to how SDKs connect to Formbricks:
+
+#### Multi-Domain Setup SDK Configuration
+
+When using PUBLIC_URL (multi-domain setup), all SDK configurations must use the **public domain**:
+
+```js
+// JavaScript/React/Vue.js/Next.js SDKs
+formbricks.setup({
+  environmentId: "your-environment-id",
+  appUrl: "https://surveys.example.com", // Use PUBLIC_URL domain
+});
+```
 
-- The `SURVEY_URL` must be a valid URL with protocol
-- When `SURVEY_URL` is set, all public survey links will use this domain.
-- If `SURVEY_URL` is not set, surveys will be served from the `WEBAPP_URL` domain
-- Make sure your DNS and SSL certificates are properly configured for both domains
-- The survey domain should be properly configured in your reverse proxy or web server
-- **Important**: The second domain is exclusively for serving surveys. Any non-survey paths accessed on this domain will return a 404 error. This is a security feature to ensure the survey domain only serves survey content.
+```jsx
+// React Native SDK
+await formbricks.setup({
+  environmentId: "your-environment-id",
+  appUrl: "https://surveys.example.com", // Use PUBLIC_URL domain
+});
+```
+
+```swift
+// iOS Swift SDK
+FormbricksSDK.shared.configure(
+  appUrl: "https://surveys.example.com", // Use PUBLIC_URL domain
+  environmentId: "yourEnvironmentId"
+)
+```
+
+#### Single Domain Setup SDK Configuration
+
+When PUBLIC_URL is not configured (single domain setup), SDKs use the admin domain:
+
+```js
+formbricks.setup({
+  environmentId: "your-environment-id",
+  appUrl: "https://formbricks.example.com", // Use WEBAPP_URL domain
+});
+```
+
+#### Key SDK Changes Made
```

Comment:
Where is this block coming from? For timeless documentation there shouldn't be "changes made" but instead just the current state how Formbricks can be configured.
I think we can remove this.